client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:3001",
+  "proxy": "http://api:3001",
```

Comment:
does this work?

Reply:
Sure does. Each Docker Compose service is addressable by its name.